src/datalad_installer.py

```diff
@@ -2617,7 +2647,6 @@ def download_file(
             return
         except URLError as e:
             if isinstance(e, HTTPError) and e.code not in (500, 502, 503, 504):
-                raise_for_ratelimit(e, headers.get("Authorization"))
```

Comment:
so we are sacrificing limits reporting here?  or it was not needed?

Reply:
ah, ok, this one is generic and not specific to github.  notes: There is a use of `download_file` within GithubClient at https://github.com/datalad/datalad-installer/blob/HEAD/src/datalad_installer.py#L2544 within `download_release_asset` but it is likely the generic url so should not get affected by the API rate limit.